packages/react/src/modules/slack/hooks/useManageSlackConnection.ts

```diff
@@ -0,0 +1,146 @@
+import { useKnockClient } from "@knocklabs/react-core";
+import { useState, useEffect, useCallback } from "react";
+
+const SLACK_AUTHORIZE_URL = "https://slack.com/oauth/v2/authorize";
+const TENANT_OBJECT_COLLECTION = "$tenants";
```

Comment:
Nit: would consider moving to a constants file if we're using this in multiple places

Reply:
will do if we reuse these but i might have it scoped to just this file if we can reuse this hook when we introduce another component that uses it